lib/lita/handlers/jenkins.rb

```diff
@@ -35,11 +35,11 @@ def self.default_config(config)
       def jenkins_build(response)
         requested_job = response.matches.last.last
 
-        if requested_job.number?
-          job = jobs[requested_job.to_i - 1]
-        else
-          job = jobs.select { |j| j['name'] == requested_job }.last
-        end
+        job = if requested_job.number?
+                jobs[requested_job.to_i - 1]
+              else
+                jobs.select { |j| j['name'] == requested_job }.last
+              end
```

Comment:
@daniely I agree completely! I try to constrain use of the ternary for _extremely simple_ logic. Glad we're on the same page here.